Backend/RoomMe/RoomMe.API/Controllers/ScheduleController.cs

```diff
@@ -0,0 +1,118 @@
+
+        [HttpGet("{scheduleId}/full", Name = nameof(GetScheduleFull))]
+        public async Task<ActionResult<ScheduleFullGetModel>> GetScheduleFull(int scheduleId)
+        {
+            var schedule = await _sqlContext.HouseworkSchedules
+                .Include(x => x.User)
+                .Include(x => x.HouseworkStatus)
+                .Include(x => x.Housework)
+                .ThenInclude(y => y.HouseworkSettings)
+                .ThenInclude(z => z.Frequency)
+                .FirstOrDefaultAsync(x => x.Id == scheduleId)
+                .ConfigureAwait(false);
+                
+
+            if(schedule == null)
+            {
+                _logger.LogError($"Schedule not found for id {scheduleId}");
+                return new BadRequestResult();
+            }
+
+            if(schedule.HouseworkStatus == null)
+            {
+                _logger.LogError($"Status not found for scheduleID {scheduleId}");
+                return new BadRequestResult();
+            }
+
+            return schedule.ToScheduleFullGetModel();
+        }
+
+        [HttpPost("", Name = nameof(CreateNewSchedule))]
+        public async Task<ActionResult<SchedulePutReturnModel>> CreateNewSchedule(SchedulePutModel schedule)
+        {
+            var housework = await _sqlContext.Houseworks.FindAsync(schedule.HouseworkId).ConfigureAwait(false);
+
+            if(housework == null)
+            {
+                _logger.LogError($"Housework not found for id {schedule.HouseworkId}");
+                return new BadRequestResult();
+            }
+
+            var scheduleEntity = schedule.ToScheduleModel(housework);
+
+            scheduleEntity.StatusId = 1;
+
+            var status = await _sqlContext.HouseworkStatuses.FindAsync(1).ConfigureAwait(false);
+
+            scheduleEntity.HouseworkStatus = status;
+
+            await _sqlContext.HouseworkSchedules.AddAsync(scheduleEntity).ConfigureAwait(false);
+            await _sqlContext.SaveChangesAsync().ConfigureAwait(false);
+
+            return scheduleEntity.ToSchedulePutReturnModel();
+        }
+
+        [HttpGet("/date", Name = nameof(GetScheduleDate))]
+        public async Task<ActionResult<List<ScheduleDateModel>>> GetScheduleDate([FromQuery] FromToDateModel dates)
+        {
+            List<ScheduleDateModel> schedulesModels = new List<ScheduleDateModel>();
+
+            var schedules = await _sqlContext.HouseworkSchedules
+                .Where(x => x.Date >= dates.From && x.Date <= dates.To)
+                .Include(y => y.Housework)
+                .ToListAsync()
+                .ConfigureAwait(false);
+
+            schedulesModels = schedules.Select(x => x.ToScheduleDateModel()).ToList();
+
+            return schedulesModels;
+        }
+
+        [HttpGet("{houseworkId}/list", Name = nameof(GetFullSchedulesByDate))]
+        public async Task<ActionResult<List<ScheduleFullGetModel>>> GetFullSchedulesByDate([FromQuery] FromToDateModel dates, int houseworkId)
```

Comment:
Tu chciałbym, zebysmy zwracali uproszczone info zamiast całego, wyobrazam sobie ze to ma nam posluzyc do tworzenia odpowiednich obiektow do kalendarza (na pewno Id, nazwa, data, status i moze id i nazwa uzytkownika)